core/src/main/java/module-info.java

```diff
@@ -172,18 +172,11 @@
   exports org.mule.runtime.core.privileged.security.tls to
       org.mule.runtime.tls;
   exports org.mule.runtime.core.privileged.transaction;
-  exports org.mule.runtime.core.privileged.transaction.xa to
-      org.mule.runtime.spring.config,
-      com.mulesoft.mule.runtime.xa,
-      com.mulesoft.mule.runtime.bti;
+  exports org.mule.runtime.core.privileged.transaction.xa;
   exports org.mule.runtime.core.privileged.transformer to
       org.mule.runtime.spring.config,
       spring.beans;
-  exports org.mule.runtime.core.privileged.util to
-      org.mule.runtime.core.components,
-      org.mule.runtime.extensions.support,
-      com.mulesoft.mule.runtime.core.ee,
-      com.mulesoft.mule.runtime.cache;
+  exports org.mule.runtime.core.privileged.util;
```

Comment:
isn't this overly optimistic? Isn't the corresondign declaration in module properites file way more restrictive?